modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -58,9 +58,11 @@
       it 'renders a list of storages' do
         visit admin_settings_storages_path
 
+        expect(page).to be_axe_clean.within '#content'
+
         expect(page).to have_test_selector('storage-name', text: complete_storage.name)
```

Comment:
Good point, will replace this with localised `test-selector` 👍🏾

Reply:
Would `have_link` not do a better job of what is trying to be accomplished?

Reply:
> Would `have_link` not do a better job of what is trying to be accomplished?

Good idea, it might- but we had a second look and Christophe proposed a really nice rewrite. Will ping once it's up 👍🏾